apps/mobile/src/components/ChatDetailScreen/components/TextMessage.tsx

```diff
@@ -87,6 +88,39 @@ function useIsExtended(messageItem: MessagesListItem): {
   }
 }
 
+type messageTypesWithItalicPrefix =
+  | 'REQUEST_MESSAGING'
+  | 'CANCEL_REQUEST_MESSAGING'
+  | 'DISAPPROVE_MESSAGING'
+  | 'APPROVE_MESSAGING'
+
+function shouldHaveItalicPrefix(
+  messageType: string
+): messageType is messageTypesWithItalicPrefix {
+  return [
+    'REQUEST_MESSAGING',
+    'CANCEL_REQUEST_MESSAGING',
+    'DISAPPROVE_MESSAGING',
+    'APPROVE_MESSAGING',
+  ].includes(messageType)
+}
+
+function TextMessageAccent({message}: {message: ChatMessage}): JSX.Element {
+  const {t} = useTranslation()
+
+  if (shouldHaveItalicPrefix(message.messageType)) {
```

Comment:
I'm proud of this 👯 . This will make sure we can use messages.textMessageTypes.${message.messageType} with only subset of MessageType while still keeping the linter on :)